### ISSUE TITLE
Select which tab groups to save

### ISSUE THREAD

**Issue by catempire** [Quicksaver/Tab-Groups#77]
I have lots of tabs in lots of tab groups. I use them like visual bookmarks to keep tabs of all the things I'm currently working on. Only I don't necessarily want to load every tab group every time I start firefox.

My suggestion is that, just like you can choose which tabs and groups to load, you should also be able to choose which tabs and groups to save. So when I am working on code, I can load all the tabs related to that, and then at the end save only that group back to disk. Same as when I am looking at home decorating (or whatever else) - I can load just those tabs into a new group, update that group of tabs, and then save that group of tabs without affecting any other tab groups or windows I may have open for other tasks. It looks like you already have all the widgets you need for this, you just need to enable the tab picker window for saving as well as loading.

When you think about tab groups like this, they become like visual bookmark groups, so maybe I can do other operations on them also. Like I can merge 2 tab group files without opening them, or I can delete tabs from a file on disk without opening them all, or things like that. I think something like this would be a much better way of bookmarking (which pretty much sucks in its current form when you have a lot of URLs in lots of different areas).

The other thing that would useful is if there was a retained preference that remembers the folder I load/save tab group files to, rather than defaulting to the last folder I downloaded some content to from one of my open tabs.

**Comment by Quicksaver** [comment]
I assume you are talking about the backup and restore feature in the add-on's options. I came up with the same idea just yesterday from a related suggestion by another user. I'll see what I can do about it.

**Comment by catempire** [comment]
Yes that's right. It would be awesome if you could get this working. As I mentioned, then this feature is more than just backup/restore - it's more like tab persistence or import/export or load/save tab groups.

**Comment by maverick74** [comment]
I kind of like bookmarks, actually. And i use tab groups, not as bookmarks but as kind of temporary-bookmarks - Things i'm working on but that i don't need to bookmark! (Usually i bookmark top domains only).

Anyway it's a great idea. The option to remember the save folder is good! Maybe we could set a folder where to save/load groups from in configs and always work whit that folder (don't know if this is already implemented...).

Would also be good to have an auto-save for already saved/archived groups as well as punctual save

Another idea would be that of an always-present-list-of-available-groups-to-load to be shown in the Backup/restore page, allowing the user to just click and load a specific group. I believe this list would be a lot more intuitive than the actual implementation
